subjects/DPxAI/colorful-legs/README.md

```diff
@@ -0,0 +1,50 @@
+## Colorful legs
+
+> JSPowered Mode
+
+### Context
+
+Your robot is cool! but you can make it even more striking! Or at least, make it show off its colorful legs to everyone!
```

Comment:
`but` should start with a capital letter

Reply:
Fixed.